guides/guide-for-adding-windows-node.md

```diff
@@ -90,20 +95,88 @@ kube-system kube-flannel-ds-54954 1/1 Running 0 1m
 Now you can add Windows-compatible versions of Flannel and kube-proxy. In order to ensure that you get a compatible version of kube-proxy, you'll need to substitute the tag of the image. The following example shows usage for Kubernetes v1.24.3, but you should adjust the version for your own deployment.
 
 ```bash
-curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/v1.24.3/g' | kubectl apply -f -
-kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
+# Make sure to modify the version of kube-proxy image to match your kubernetes version
+# before applying it
+wget https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/flannel/kube-proxy/kube-proxy.yml
+kubectl apply -f sig-windows-tools/hostprocess/flannel/kube-proxy/kube-proxy.yml
```

Comment:
wget downloads this file as `kube-proxy.yml` not the full path.  We should apply that file or use the SED trick used in the original.